app/src/main/java/org/wikipedia/yearinreview/YearInReviewHighlightsScreen.kt

```diff
@@ -0,0 +1,320 @@
+import org.wikipedia.compose.ComposeColors
+import org.wikipedia.compose.theme.BaseTheme
+import org.wikipedia.compose.theme.WikipediaTheme
+import org.wikipedia.theme.Theme
+
+@Composable
+fun YearInReviewHighlightsScreen(
+    modifier: Modifier = Modifier,
+    screenData: YearInReviewScreenData.HighlightsScreen,
+    onShareHighlights: () -> Unit
+) {
+    Box(modifier = Modifier.fillMaxSize()) {
+        Column(
+            modifier = modifier
+                .yearInReviewHeaderBackground()
+                .fillMaxSize()
+                .padding(horizontal = 18.dp)
+        ) {
+            Text(
+                modifier = Modifier
+                    .padding(top = 60.dp, bottom = 16.dp),
+                text = buildAnnotatedString {
+                    append(stringResource(R.string.year_in_review_highlights_thank_you_message))
+                    withStyle(
+                        style = SpanStyle(
+                            fontWeight = FontWeight.Normal,
+                        )
+                    ) {
+                        append(" ")
+                        append(stringResource(R.string.year_in_review_highlights_looking_forward_message))
+                    }
+                },
+                color = WikipediaTheme.colors.paperColor,
+                style = MaterialTheme.typography.bodyLarge,
+                textAlign = TextAlign.Center,
+                fontWeight = FontWeight.Medium
+            )
+
+            ShareableHighlightsCard(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(324.dp)
+                    .background(ComposeColors.Gray100)
+                    .border(width = 1.dp, color = ComposeColors.Gray300)
+                    .padding(8.dp)
+                    .verticalScroll(rememberScrollState()),
+                highlights = screenData.highlights
+            )
+
+            Button(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 32.dp),
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = WikipediaTheme.colors.progressiveColor
+                ),
+                onClick = onShareHighlights
+            ) {
+                Text(
+                    "Share highlights"
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun ShareableHighlightsCard(
+    modifier: Modifier = Modifier,
+    hashtag: String = "#WikipediaYearinReview",
+    logoResource: Int = R.drawable.w_nav_mark,
+    logoDescription: String = "Wikipedia logo",
+    highlights: List<YearInReviewScreenData.HighlightItem>,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = hashtag,
+            fontWeight = FontWeight.Bold,
+            fontSize = 18.sp,
+            lineHeight = 21.sp,
+            color = ComposeColors.Gray700
```

Comment:
Not sure if you asked whether the background/text should fit into different themes.